src/Cmfd.cpp

```diff
@@ -1456,6 +1457,11 @@ void Cmfd::initializeMaterials() {
     delete [] _materials;
   }
 
+  /* Compute and log size in memory of Material array */
+  double size = (double) (_num_cmfd_groups + 4) * _num_cmfd_groups *
+              _local_num_xn * _local_num_yn * _local_num_zn * 
+              sizeof(FP_PRECISION) / (double) 1e6;
+  log_printf(NORMAL, "CMFD material storage per domain = %6.2f MB", size);
```

Comment:
For line 1472. For BEAVRS assembly, axially 230, radially 289, the total number is 66,470 while DEFAULT_INIT_ID is 100,000. Kind of close. Suggest increase DEFAULT_INIT_ID .